examples/README.md

```diff
@@ -54,3 +54,48 @@ aws s3api create-bucket --bucket <bucket-name> --region <region>
   <ecs-task-family> \
   <ecs-service-name>
 ```  
+
+## FireLens on ECS Fargate
+
+For running on Fargate, s3 file option is not supported for Fluent-bit config. Hence, we need to build the fluent-bit custom config image and then use its reference in our Firelens container definition.
+
+For building the custom fluent-bit image, clone the [amazon-ecs-firelens-examples](https://github.com/aws-samples/amazon-ecs-firelens-examples) and modify the contents of [extra.conf](https://github.com/aws-samples/amazon-ecs-firelens-examples/blob/mainline/examples/fluent-bit/config-file-type-file/extra.conf) in the amazon-ecs-firelens-examples repository. Post this run the following commands to build the custom fluent-bit image:-
+
+```sh 
+# create an ECR repository for the Fluentbit-config image
+aws ecr create-repository --repository-name <config-image-name> --region <region>
+
+# Navigate to the config file directory
+cd examples/fluent-bit/config-file-type-file
+
+# Build the docker image from Dockerfile. Replace config-image-name with your image name
+docker build -t <config-image-name> .
+
+# Tag the recently built docker image . Replace the config-image-name, account-id and region with your values.
+docker tag <config-image-name>:latest <account-id>.dkr.ecr.<region>.amazonaws.com/<config-image-name>:latest
+
+# Push the docker image to ECR
+docker push <account-id>.dkr.ecr.<region>.amazonaws.com/<config-image-name>:latest
+```
+
+For executing EMF application on Fargate, you need to execute the following commands :-
+
+```sh 
+# create an ECR repository for the example image
+aws ecr create-repository --repository-name <image-name> --region <region>
+
+# create ECS cluster
+# create ECS task definition
+# create ECS service
+
+# deploy
+./examples/ecs-firelens/bin/publish-fargate.sh 
```

Comment:
have you posted change for `publish-fargate.sh` ?

Reply:
Yeah it is in the second commit of this pull request. I can see it in the files changed.
Let me know if you are unable to view the file.

Reply:
The bin folder is ignored by git that is why the file got missed in the previous commit.
I used the -f option to add it explicitly and then added another commit for that.